src/browser/ui/dom/DOMPropertyOperations.js

```diff
@@ -61,7 +61,7 @@ if (__DEV__) {
     // logging too much when using transferPropsTo.
     warning(
       standardName == null,
-      'Unknown DOM property ' + name + '. Did you mean ' + standardName + '?'
+      'Unknown DOM property ' + name + (value == null ? '': ', value: ' + value) + '. Did you mean ' + standardName + '?'
```

Comment:
Lint is complaining about formatting in here. Let's fix that. It might be easier to actually just build and assign that value string outside of this, otherwise it'll get annoying to read.

Let's also tweak this to match other cases (where we use \` to indicate a code value, like markdown):
> Unknown DOM property `class` (with value `foo`). Did you mean `className`?